src/ec_filter.c

```diff
@@ -1085,7 +1085,13 @@ int filter_load_file(const char *filename, struct filter_list **list, uint8_t en
 
    /* sanity checks */
    if (fh.magic != htons(EC_FILTER_MAGIC))
-      FATAL_MSG("Bad magic in filter file\nMake sure to compile the filter with etterfilter");
+      FATAL_MSG("Bad magic in filter file\n"
+            "Make sure to compile the filter with a current version of etterfilter");
+
+   /* check pointer alignment */
+   if (fh.code % 8)
+      FATAL_MSG("Bad instruction pointer alignment\n"
+            "Make sure to compile the filter with a current version of etterfilter");
```

Comment:
The wording is not clear. The _current_ version is the last released version, which obviously still has this issue.

Reply:
Do you have a suggestion? 

I tried to formulate in a way that doesn’t require frequent adjustments or confuses even more.

Maybe I should additionally drop a note in the CHANGES file which acts as a sort of release notes.

What do you think?

Reply:
A note in CHANGES or even README is a good idea. This is a big change after all.
A git tag may be helpful as well.